src/Configuration/src/SpringBootCore/Steeltoe.Extensions.Configuration.SpringBootCore.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\PlaceholderCore\Steeltoe.Extensions.Configuration.PlaceholderCore.csproj" />
```

Comment:
Why was this dependency added? It looks like only the _test_ project needs it.

Reply:
Yes, fixed it.